source/MaterialXTest/Runtime.cpp

```diff
@@ -1180,6 +1180,14 @@ TEST_CASE("Runtime: Looks", "[runtime]")
     assign2.getExclusive().getValue().asBool() = false;
     look1.addMaterialAssign(pa);
     look1.addMaterialAssign(pa2);
+    try
+    {
+        look1.addMaterialAssign(col1.getPrim());
+    }
+    catch (mx::ExceptionRuntimeError&)
+    {
+        REQUIRE(true);
+    }
```

Comment:
This will not generate a test error if no exception is thrown. I think you want to replace the whole block here with: 
`REQUIRE_THROWS(look1.addMaterialAssign(col1.getPrim()));`

Reply:
Much better. Thanks.